content/oci/oci.go

```diff
@@ -526,6 +512,38 @@ func (s *Store) GC(ctx context.Context) error {
 	return nil
 }
 
+// gcIndex reloads the index and updates metadata.
```

Comment:
nit: Should we improve this comment a bit? Like mentioning that we clean up the untagged descriptors and leave tagged ones.

Reply:
Improved the doc.